components/html-block/demo/html-block.html

```diff
@@ -10,6 +10,18 @@
 			import '../../demo/demo-page.js';
 			import '../html-block.js';
 		</script>
+		<script>
+			if (window.location.search.indexOf('latex=true') !== -1) {
+				/* eslint-disable-next-line no-global-assign */
```

Comment:
Linting thinks that `D2L` is a global readonly variable... Which is generally true, but not on demo pages.

Reply:
I think you can also avoid the linting error by starting with `window.`... i.e. `window.D2L = {};`

Reply:
Ah, I'll try that then. Better than adding linting overrides.